src/components/RecNode/RecNode.tsx

```diff
@@ -41,10 +42,24 @@ const RecNode: FC<RecNodeProps> = ({ pathwayState, isActionable }) => {
       {isActionable && (
         <form className={classes.commentsForm}>
           <label>Comments:</label>
-          <button className={indexClasses.button} onClick={(e): void => e.preventDefault()}>
+          <button
+            className={indexClasses.button}
+            onClick={(e): void => {
+              e.preventDefault();
+              setComments(
+                comments +
+                  // eslint-disable-next-line max-len
+                  `\nThe patient and I discussed the treatment plan, risks, benefits and alternatives.  The patient expressed understanding and wants to proceed.`
```

Comment:
Why the `\n`?

![Screen Shot 2020-02-05 at 3 39 15 PM](https://user-images.githubusercontent.com/41651655/73881082-b1af3800-482d-11ea-85cc-8deb6b32ab9d.png)

Reply:
wanted to keep it separated from text already in the box, but I guess when it's alone it doesn't need it.